src/Goerli-DssSpell.t.sol

```diff
@@ -254,18 +285,19 @@ contract DssSpellTest is GoerliDssSpellTestBase {
 
         uint256 castTime = block.timestamp + pause.delay();
         hevm.warp(castTime);
-        (, , , uint48 tocPre) = oracle.ilks("RWA008AT3-A");
+        (, , , uint48 tocPre) = oracle.ilks("RWA009AT1-A");
         assertTrue(tocPre == 0);
-        assertTrue(oracle.good("RWA008AT3-A"));
+        assertTrue(oracle.good("RWA009AT1-A"));
         tellSpell.cast();
-        (, , , uint48 tocPost) = oracle.ilks("RWA008AT3-A");
+        (, , , uint48 tocPost) = oracle.ilks("RWA009AT1-A");
         assertTrue(tocPost > 0);
-        assertTrue(oracle.good("RWA008AT3-A"));
+        // should be not good as we have TAU 0
+        assertTrue(!oracle.good("RWA009AT1-A"));
         hevm.warp(block.timestamp + 2 weeks);
-        assertTrue(!oracle.good("RWA008AT3-A"));
+        assertTrue(!oracle.good("RWA009AT1-A"));
     }
 
-    function testSpellIsCast_RWA008AT3_INTEGRATION_TELL_CURE_GOOD() public {
+    function testSpellIsCast_RWA009AT1_INTEGRATION_TELL_CURE_GOOD() public {
```

Comment:
This test doesn't make sense anymore.

Its original purpose was to test whether the ilk could return to its normal state after it had entered the remediation state through `tell()`.

However, since `TAU == 0`, it means that this is not possible. This test should either be **removed** or **changed** to assert that it cannot `cure()` even if it's called immediately after `tell()`.